src/sentry/integrations/message_builder.py

```diff
@@ -19,37 +19,50 @@
 from sentry.utils.http import absolute_uri
 
 
-def format_actor_options(
-    actors: Iterable[Team | RpcUser], is_slack: bool = False
```

Comment:
it's technically possible to preserve the old functions here with `@overload` but I figured it's best to just kill the [boolean trap](https://youtu.be/CnRkXO_a5mI) rather than encourage it